aiida/orm/implementation/general/authinfo.py

```diff
@@ -116,8 +116,9 @@ def computer(self):
 
     @property
     def user(self):
-        from aiida.orm.user import User
-        return User.get(email = self.dbauthinfo.aiidauser.email)
+        # TODO: @mu FIX THIS!
```

Comment:
Did you put it in the commit message at least? How could you sleep at night otherwise...

Reply:
Woops!  Good spot.  Fixed in the latest commit.